app/src/main/java/com/skyyo/samples/application/Destination.kt

```diff
@@ -65,6 +65,7 @@ sealed class Destination(val route: String) {
     object CardRecognition : Destination("cardRecognition")
     object GooglePay : Destination("googlePay")
     object TextSpans : Destination("textSpans")
+    object LazyColumn : Destination("lazyColumn")
```

Comment:
same here and in other places, so it's not confusing to the reader. `LazyColumnWithTextField`

Reply:
ok.
I named it as base, because we can encounter some other issues with lazycolumns, so we can add fixes into this composable, and use it everywhere, no matter if there is TextField inside or there is no TextFields.
The only drawback of this composable vs standard lazycolumn that it handles "imePadding" internally, not via modifier.
Otherwise I'd insist to keep it as base, not "ime based"

Reply:
Better to be specific about issue component handles, right until we need more than 1 fix inside it. 
`BaseLazyColum` just doesn't tell the user what's the difference is in comparison with `LazyColumn`

Reply:
Ok :+1: